### PR TITLE
Navigate

### PR DESCRIPTION
**Pull request by vladd-png**
### Is This PR a:
- [x] Feature
- [ ] Bug Fix
- [ ] Documentation

### What's this PR do?
Adds dependencies for stacks to create user navigation in app
Moves login out of app component and creates new component called Login
Imports login into App
Adds stacks to app

### Where should a reviewer start?
App and Login

### Next steps or Questions
I have to figure out navigation next. How to actually move the page around.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,82 +1,19 @@
...
-      <Separator />
-      <View>
-        <TouchableOpacity style={ styles.buttonBottom } >
-          <Text style={{ fontSize: 32, color: 'lightgray' }}>Run Today</Text>
-        </TouchableOpacity>
-      </View>
-      </ImageBackground>
-    </SafeAreaView>
+    <NavigationContainer>
+      <Stack.Navigator>
```
Comment (VeeAndrade) [inline App.js:10]: This is super cool to look at! Are the components pre set?

File: Login.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { StyleSheet, Button, View, SafeAreaView, Text, Alert, TouchableOpacity, ImageBackground } from 'react-native';
```
Comment (VeeAndrade) [inline Login.js:2]: Oh I guess that answered my last question, does this limit what you're able to do at all?

File: Login.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        </TouchableOpacity>
+      </View>
+      </ImageBackground>
+    </SafeAreaView>
+    </NavigationContainer>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
```
Comment (VeeAndrade) [inline Login.js:33]: It's interesting to see the styles in the same file as your js

File: App.js (hunk 1 above)
Comment (vladd-png) [reply App.js:10]: Components are from an NPM dependency yes

File: Login.js (hunk 2 above)
Comment (vladd-png) [reply Login.js:2]: There is a limit to the preset templaets - but there are several templates to choose from

File: Login.js (hunk 3 above)
Comment (vladd-png) [reply Login.js:33]: this is what svelte does as well

**Pull request merged**